Find a functionally equivalent code:android.net.ConnectivityManager.getNetworkTypeName:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * Returns a non-localized string representing a given network type.
 * ONLY used for debugging output.
 * @param type the type needing naming
 * @return a String for the given type, or a string version of the type ("87")
 * if no name is known.
 * {@hide}
 */

@@ -30,35 +30,39 @@
             return "WIFI_P2P";
         case TYPE_MOBILE_IA:
             return "MOBILE_IA";
+        case TYPE_MOBILE_EMERGENCY:
+            return "MOBILE_EMERGENCY";
+        case TYPE_PROXY:
+            return "PROXY";
         default:
             return Integer.toString(type);
     }

